packages/world/src/systems/ActivateSystem.sol

```diff
@@ -33,9 +32,7 @@ contract ActivateSystem is System {
   }
 
   function activatePlayer(address playerAddress) public {
-    EntityId player = Player._get(playerAddress);
-    player = player.baseEntityId();
-    require(player.exists(), "Entity does not exist");
+    EntityId player = EntityIdLib.encodePlayer(playerAddress);
```

Comment:
I assume the entity id encoded with the address is always the base entity id

Reply:
yes, we use an incremental one for the relative coord. Another option is to actually use the same entity id for both, and just map the different coords to it